webapi/src/main/java/com/dumbdogdiner/stickyapi/webapi/mojang/MojangAPI.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.webapi.mojang;
+
+import com.dumbdogdiner.stickyapi.annotation.Broken;
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.dumbdogdiner.stickyapi.util.http.HttpConnectionException;
+import com.dumbdogdiner.stickyapi.util.http.HttpException;
+import com.dumbdogdiner.stickyapi.util.http.HttpStatusException;
+import com.dumbdogdiner.stickyapi.util.http.HttpUtil;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.UUID;
+
+
+// Double check the occurances of this and whatnot, try to fix, etc.
+public class MojangAPI {
+    /**
+     * When possible, use the cached, faster api at https://api.ashcon.app/mojang/v2/user, otherwise use mojang
+     * API URLs
+     */
+
+    private static final @NotNull Gson G = HttpUtil.getDefaultGsonInstance();
+    private static final @NotNull OkHttpClient CLIENT = HttpUtil.getDefaultClientInstance();
+
+
+    protected static final @NotNull HttpUrl COMBINED_API_URL = new HttpUrl.Builder()
```

Comment:
Would using strings be easier?

Reply:
No

Reply:
Why not

Reply:
Do you mean using .parse? I'm changing to that. HttpUrl is useful for setting flags

Reply:
Yes

Reply:
Did you end up changing to `.parse`?